skimage/measure/tests/test_marching_cubes.py

```diff
@@ -94,10 +95,11 @@ def test_correct_mesh_orientation():
                       [5, 4, 3]])
 
     # Correct mesh orientation - descent
-    corrected_faces1 = correct_mesh_orientation(sphere_small, verts, faces,
-                                                gradient_direction='descent')
-    corrected_faces2 = correct_mesh_orientation(sphere_small, verts, faces,
-                                                gradient_direction='ascent')
+    with expected_warnings(['`correct_mesh_orientation` is deprecated']):
```

Comment:
Hmm... It seems like we forgot to specify the version for the removal of `correct_mesh_orientation`. The earliest known related PR is from July 2016 - https://github.com/scikit-image/scikit-image/commit/a90f875d15333230ad3bfbd7f526af9a889513d5#diff-be4843b3247c5ef60c02dd4290071a0eL255.
@scikit-image/core Any objections to remove the function?

Reply:
@soupault none! Crazy stuff!